### ISSUE TITLE
data.json and dataa.json

### ISSUE THREAD

**Issue by NickP123** [FACN3/swearUrban#33]
What is dataa.json doing? Assuming was used for development before moving over to the full data.json file?

**Comment by hasansaad1** [comment]
Yes, we used it because it had fewer data, and data that is ready to be used.
deleting it right now.

**Comment by hasansaad1** [comment]
fixed @NickP123
